CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+# Next Release
+Requests will now fail fast with an error if an API key is not provided instead of making a live API call with no key.
```

Comment:
This needs a line break after the header
```suggestion
## Next Release

Requests will now fail fast with an error if an API key is not provided instead of making a live API call with no key.
```